### ISSUE TITLE
Corrupted data file on power failure

### ISSUE THREAD

**Issue by chrissound** [erebe/greenclip#61]
```
Apr 08 18:02:51 nixos greenclip[10417]: Data.Binary.Get.runGet at position 0: not enough bytes
Apr 08 18:02:51 nixos greenclip[10417]: CallStack (from HasCallStack):
Apr 08 18:02:51 nixos greenclip[10417]:   error, called at libraries/binary/src/Data/Binary/Get.hs:351:5 in binary-0.8.5.1:Data.Binary.Get
```

I've just had a power failure and it seems I've lost all my previous history. In addition I see the above error.

I think this file could be checked at runtime when starting the daemon, and to terminate with an error if it's not valid.

**Comment by D1mon** [comment]
`greenclip clear`
and update greenclip package

**Comment by erebe** [comment]
fixed in 3.3
